### PR TITLE
refactor: bump all deps versions because of vulnerabilities

### PR DESCRIPTION
**Pull request by giulianok**
The library hasn't been published in a year, and dependencies have to be upgraded for security reasons. At the moment, there are 6 vulnerabilities reported by npm, 4 of them are high

Screen Shot 2020-08-08 at 2 44 26 PM

This PR bumps all the versions and performs necessary changes required by newer dependencies' versions.

Screen Shot 2020-08-08 at 2 46 09 PM

Screen Shot 2020-08-08 at 3 06 33 PM

### CODE REVIEW

**Comment by giulianok** [comment]
reported in https://github.com/yeoman/insight/issues/71

**Comment by sindresorhus** [comment]
For each dependency with a major bump, you need to go through their release notes and migrate accordingly.

File: .npmrc [hunk 1]
```diff
@@ -1 +0,0 @@
-package-lock=false
```
Comment (sindresorhus) [inline .npmrc:1]: Don't do unrelated changes.

Comment (giulianok) [reply .npmrc:1]: this is required to run `npm audit` and npm recommends committing it

**Comment by giulianok** [comment]
@sindresorhus 

> you need to go through their release notes and migrate accordingly

can you be more specific? perhaps provide an example? I would like to understand the scope of the work and see if multiple PRs are needed.

> Just the fact that the tests passes are not enough.

Are you saying that I should run tests + `go through their release notes` in order to guarantee bug-free update? I believe there's a missing piece here, which is integration tests and manual tests. Assistance on manual tests would be great

**Comment by sindresorhus** [comment]
Tests cannot catch everything, no matter how comprehensive. You need to go check for breaking changes in the dependencies you have bumped.

**Comment by raphinesse** [comment]
For example: updating conf did break a lot last time I tried. See #69.

**Comment by giulianok** [comment]
I agree. At the moment, i'm short on time, has some personal things going on (one of them a water pipe broke in my apt and i'm literally homeless at the moment lol)

What i want to do is
- Check the changelogs and try to find breaking changes. This might be a bit challenging since the deps are several major versions behind.
- Run manual tests to make sure everything works as expected.

I've never used this library, so if anyone that's actively using can give a bit of help on testing it would be great

**Comment by sindresorhus** [comment]
Closing for lack of activity.
